Tests/SnapshotTestingTests/__Snapshots__/SnapshotTestingTests/testNSView.2.txt

```diff
@@ -0,0 +1,4 @@
+[   AF      LU ] h=--- v=--- NSButton "Push Me" f=(0,0,77,32) b=(-)
```

Comment:
NSView's version of `recursiveDescription` is `_subtreeDescription`, which misses the main view. Prob wanna:

- [ ] Embed the view in a superview before calling this method on the superview.